chromium_src/components/variations/service/field_trial_unittest.cc

```diff
@@ -14,7 +14,7 @@ TEST_F(FieldTrialsTest, FieldTrialsTestingEnabled) {
 #if BUILDFLAG(FIELDTRIAL_TESTING_ENABLED)
   enabled = true;
 #endif  // BUILDFLAG(FIELDTRIAL_TESTING_ENABLED)
-  EXPECT_FALSE(enabled);
+  EXPECT_TRUE(enabled);
```

Comment:
see https://github.com/brave/brave-core/pull/6940/files#r519220155 - this should still skip using the testing config because we're setting `kVariationsServerURL`, but I don't see why we'd explicitly enable this when we don't want it

Reply:
reverted